ddtrace/debugging/_debugger.py

```diff
@@ -245,6 +247,15 @@ def __init__(self, tracer=None):
             raise_on_exceed=False,
         )
 
+        # TODO: this is only temporary and will be reverted once the DD_REMOTE_CONFIGURATION_ENABLED variable
```

Comment:
aside: will there be no way to disable remote config in the future?

Reply:
this is only needed so long as the default of DD_REMOTE_CONFIGURATION_ENABLED is false, right? Once the default is true then we don't need to do this